HarpGuru/HarpFace/HarpCell/components/index.test.tsx

```diff
@@ -10,9 +11,12 @@ import { DisplayModes } from '../../../types'
 
 import { HarpCell } from './index'
 
+jest.mock('reactn')
+const mockUseGlobal = useGlobal as jest.Mock
+
 test('A component is rendered with the Degree or Pitch value in its text view depending on the DisplayMode selected', () => {
+  mockUseGlobal.mockReturnValue([inactiveCellsHarpFaceProps.activeHarpStrata])
```

Comment:
Now that we have fragmented the dependencies from just being parameter based to now being partially global state based, we have more reason than ever to break up these testResources so that there is one producing HarpStratas  and that is composed together with some other ones in the various test situations.

This may need to be done in a different pull request, and I will in fact put a link to this comment in https://github.com/js-jslog/harpnative/issues/28 to identify that this is a consideration across the board now.

Reply:
Comment made at https://github.com/js-jslog/harpnative/issues/28#issuecomment-660520265